cli/src/templates/packages/expo-router/drawer/app/(drawer)/index.tsx.ejs

```diff
@@ -0,0 +1,10 @@
+import { View, Text } from 'react-native';
+
+const Page = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```

Comment:
Can you add a conditional here that changes this based on the `stylingPackage` selected? (stylesheet, nativewind, tamagui)